lib/DataQueue/src/DataQueue.cpp

```diff
@@ -0,0 +1,31 @@
+#include "DataQueue.h"
+
+DataQueue::DataQueue() {
+	SYSTEM_THREAD(ENABLED);
```

Comment:
We investigated the use of this statement, and came to the conclusion that this _has_ to be called before your application code (so before `setup()` and `loop()`). In this case, it'd make sense to just make `SYSTEM_THREAD(ENABLED)` a hard requirement for this library, or it will fail in unpredictable ways.